02.calendar/calendar.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'date'
+require 'optparse'
+
+class Calender
+  def initialize(year: Date.today.year, month: Date.today.month)
+    @year = year
+    @month = month
+  end
+
+  def calender
+    start_month = Date.new(@year, @month, 1)
+    end_month = Date.new(@year, @month, -1)
+
+    puts "#{@month}月  #{@year}".center(20)
+
+    puts " 日 月 火 水 木 金 土"
+
+    (start_month.wday * 3).times { print " " }
```

Comment:
Rubyだとこんな書き方もできますねー。

```suggestion
    print " " * start_month.wday * 3
```